### ISSUE TITLE
[Bug]: Origin tauri://localhost is not allowed by Access-Control-Allow-Origin. Status code: 204

### ISSUE THREAD

**Issue by vibl** [Ironclad/rivet#153]
### What happened?

I get a CORS error when I send an HTTP request (same error from browser or Node.js).

```
[Error] Origin tauri://localhost is not allowed by Access-Control-Allow-Origin. Status code: 204
[Error] Fetch API cannot load https://api-prod.omnivore.app/api/graphql due to access control checks.
```

The same query works from Postman and curl.

### What was the expected functionality?

HTTP

### Describe your environment

Kubuntu 23.04
rivet_1.2.1_amd64.AppImage

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vibl** [comment]
It does when the executor is set on "Node", but I now realize that the popup "Remote Debugger (Reconnecting...)" stays up, which probably mean that the executor has not really switched to Node.

Also, when it is set to "Node", the Executor dropdown flickers and there are flashes of the word "Remote" appearing for a 100ms or so (alternating with "Node").

**Comment by abrenneke** [comment]
Yeah the UI for the node executor needs to be improved, definitely. If it never actually connects, then you're right, the node executor isn't starting successfully. Are you running from source or from a compiled version?

**Comment by abrenneke** [comment]
If you're running from source, you can actually run the node executor and connect to it manually:

* cd to `packages/app-executor` and run `yarn start --port 21900`
* Connect the Rivet remote debugger to `ws://localhost:21900`
* You'll then get full lots where you started the executor, in case something is going wrong there.

**Comment by vibl** [comment]
I'm running from the compiled version but I will try from source.
